ui/src/js/viewModels/orgAdmin.js

```diff
@@ -7,436 +7,453 @@
-                                "admin": (self.userAdmin().length > 0) ? "Y" : "N",
-                                "need_approver": (self.userNeedApprover().length > 0) ? "Y" : "N",
-                                "user_approver": (self.userApprover().length > 0) ? "Y" : "N",
-                                "manage_clients": (self.userManageClients().length > 0) ? "Y" : "N",
-                                "manage_offers": (self.userManageOffers().length > 0) ? "Y" : "N",
-                                "client_share_approver": (self.userClientShareApprover().length > 0) ? "Y" : "N",
-                                "confirmed": (self.userConfirmed().length > 0) ? "Y" : "N"
-                            };
-                        return $.when(restClient.doPost('/rest/users', userData)
-                            .then(
-                                success = function (response) {
-                                    self.postText("You have succesfully saved user details.");
-                                    self.postTextColor("green");
-                                    return $.when(restClient.doPost('/rest/organizations/' + self.userOrgId() + '/user_organizations', userOrgData)
-                                        .then(
-                                            success = function (response) {
-                                                self.getUserOrgData(self.userOrgId());
-                                                console.log("user data posted");
-                                            })
-                                    );
-                                },
-                                error = function (response) {
-                                    self.postText("Error: User changes not saved.");
-                                    self.postTextColor("red");
-                                    console.log("user data not posted");
-                                }).then(function () {
+                    /*
+                    var userOrgData =
+                    {
+                        "id": self.id(),
+                        "user_id": self.userId(),
+                        "organization_id": self.userOrgId(),
+                        "admin": (self.userAdmin().length > 0) ? "Y" : "N",
+                        "need_approver": (self.userNeedApprover().length > 0) ? "Y" : "N",
+                        "user_approver": (self.userApprover().length > 0) ? "Y" : "N",
+                        "manage_clients": (self.userManageClients().length > 0) ? "Y" : "N",
+                        "manage_offers": (self.userManageOffers().length > 0) ? "Y" : "N",
+                        "client_share_approver": (self.userClientShareApprover().length > 0) ? "Y" : "N",
+                        "dbs_check": (self.userDbsCheck()) ? "P" : "P",
+                        "confirmed": (self.userConfirmed().length > 0) ? "Y" : "N"
+                    };
+                    */
```

Comment:
This variable "userOrgData" used to be later on in the function, but that bit has since been commented out, so this variable wasn't being used at all, therefore I thought I would comment out this variable being declared since it isn't used. The function getUserOrgData is used instead of this variable.